app/src/main/kotlin/com/backbase/android/business/journey/common/user/UserRepository.kt

```diff
@@ -0,0 +1,83 @@
+package com.backbase.android.business.journey.common.user
```

Comment:
I would recommend to delete the unused functionality or code  since we want to minimize the code that is actually needed to set up the golden sample app